### ISSUE TITLE
Create an error list

### ISSUE THREAD

**Issue by VictorVelarde** [CartoDB/carto-vl#963]
### Related to: https://github.com/CartoDB/carto-vl/issues/743

It might be useful to create a list with all the errors in the library. A previous refactoring has been done, regarding to error types and messages composition (see https://github.com/CartoDB/carto-vl/pull/924).

**Comment by davidmanzanares** [comment]
I think #924 closed this, right?

**Comment by VictorVelarde** [comment]
Not exactly. The merged commits were a refactoring to organize error types and their messages. 

This is the pending request to extract a list from the code itself, using a tool like [espree](https://github.com/eslint/espree), described in https://github.com/CartoDB/carto-vl/pull/924#issuecomment-419391239
